lib/utils/sa-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { Cluster, ServiceAccount } from "@aws-cdk/aws-eks";
+import { PolicyDocument, ManagedPolicy } from "@aws-cdk/aws-iam";
+
+/**
+ * Creates a service account that can access secrets
+ * @param clusterInfo 
+ * @returns sa
+ */
+export function createServiceAccount(cluster: Cluster, name: string, namespace: string, policyDocument: PolicyDocument): ServiceAccount {
+    const sa = cluster.addServiceAccount(`${name}-sa`, {
+        name: name,
+        namespace: namespace,
+    });
+
+    const policy = new ManagedPolicy(cluster, `${name}-managed-policy`, {
+        document: policyDocument
+    });
+
+    sa.role.addManagedPolicy(policy)
```

Comment:
We  use google coding style guide for TS. So let's make sure we consistently terminate lines with ";" based on  https://google.github.io/styleguide/tsguide.html#automatic-semicolon-insertion

Reply:
I will enforce semi on ESLint and apply changes accordingly.